### DISCUSSION TITLE
Documentation page not working

### DISCUSSION THREAD

**Discussion by vivek-yes** [JustFly1984/react-google-maps-api#3267] [General]
I am trying to access this page and getting some error https://react-google-maps-api-docs.netlify.app/

**Comment by chris-iliopoulos** [comment]
Same here, I've been trying for a whole week!

**Reply by carlosag012** [reply]
I've been using this library for a while now and the documentation was very useful. However, the other day I was trying to refresh my memory and tried to access it but to my surprise, it wasn't there anymore in NPM. Here it is a very close documentation, I believe this library is using react-goolge-maps as dependency [https://tomchentw.github.io/react-google-maps/#introduction](url)

**Reply by penguinuux** [reply]
I am using the documentation from this web archives link [https://web.archive.org/web/20230701010714mp_/https://react-google-maps-api-docs.netlify.app/#section-introduction](https://web.archive.org/web/20230701010714mp_/https://react-google-maps-api-docs.netlify.app/#section-introduction). Works well!
